region/maridia/outer.json

```diff
@@ -6524,6 +6971,20 @@
                     "Only requires a runway of approximately 6 tiles in the adjacent room.",
                     "Perform an air ball while against the ceiling to make it up and through the morph tunnel."
                   ]
+                },
+                {
+                  "name": "G-Mode Morph IBJ",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2,3],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
+                    "Gravity"
+                  ],
+                  "note": "Kill or avoid the crabs."
```

Comment:
not trivial

Reply:
Doing the strat right away worked for avoiding crabs.  Fast crab is kinda hard to kill, I wanted two passes for that one.